test/client/fixtures/runner/automation/select-element-test.js

```diff
@@ -591,139 +630,164 @@ $(document).ready(function () {
+                $option = $(select).children(':first');
 
-        select.selectedIndex = 0;
-        equal(select.selectedIndex, 0);
-        clickPlaybackAutomation($(select).children()[3], {}, function () {
-            equal(select.selectedIndex, 3);
-            startNext();
+            $option.attr('disabled', 'disabled');
+
+            select.selectedIndex = 0;
+            equal(select.selectedIndex, 0);
+            clickPlaybackAutomation($option[0], {}, function () {
+                equal(select.selectedIndex, 0);
+                startNext();
+            });
         });
-    });
 
+        asyncTest('click on option (select size more than options count)', function () {
+            var select = createSelect(10);
 
-    module('press actions with select element with attribute size more than one');
-    asyncTest('press down/up in select', function () {
-        var select = createSelect();
+            select.selectedIndex = 0;
+            equal(select.selectedIndex, 0);
+            clickPlaybackAutomation($(select).children()[3], {}, function () {
+                equal(select.selectedIndex, 3);
+                startNext();
+            });
+        });
 
-        $(select).attr('size', '2');
-        //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
-        select.selectedIndex = 0;
-        $(select).focus();
+        module('press actions with select element with attribute size more than one');
+        asyncTest('press down/up in select', function () {
+            var select = createSelect();
 
-        pressDownUpKeysActions(select);
-    });
+            $(select).attr('size', '2');
+            //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
+            select.selectedIndex = 0;
+            $(select).focus();
 
-    asyncTest('press right/left in select', function () {
-        var select = createSelect();
+            pressDownUpKeysActions(select);
+        });
 
-        $(select).attr('size', '2');
-        //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
-        select.selectedIndex = 0;
-        $(select).focus();
+        asyncTest('press right/left in select', function () {
+            var select = createSelect();
 
-        pressRightLeftKeysActions(select);
-    });
+            $(select).attr('size', '2');
+            //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
+            select.selectedIndex = 0;
+            $(select).focus();
 
-    asyncTest('press down/up in select with "size" more than options count', function () {
-        var select = createSelect();
+            pressRightLeftKeysActions(select);
+        });
 
-        $(select).attr('size', '10');
-        //NOTE: IE11 set selectedIndex = -1 after setting attribute size != 1
-        select.selectedIndex = 0;
-        $(select).focus();
+        asyncTest('press down/up in select with "size" more than options count', function () {
```

Comment:
asyncTest('press down/up in select with "size" more than the option count', function () {